cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/GoalViolationDetector.java

```diff
@@ -251,9 +258,9 @@ protected boolean optimizeForGoal(ClusterModel clusterModel,
       goal.optimize(clusterModel,
                     new HashSet<>(),
                     _optimizationOptionsGenerator.optimizationOptionsForGoalViolationDetection(clusterModel,
-                                                                                               excludedTopics(clusterModel),
-                                                                                               excludedBrokersForLeadership,
-                                                                                               excludedBrokersForReplicaMove));
+                        excludedTopics(clusterModel),
+                        excludedBrokersForLeadership,
+                        excludedBrokersForReplicaMove));
```

Comment:
Why are we breaking this indentation?

Can we please use the project's style file: https://github.com/linkedin/cruise-control/blob/master/docs/code-style.xml ?